CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+- Fix keyboard navigation (#375)
```

Comment:
Could you add a header here of `### Fixed`

So it'd be:
```md
## [Unreleased]

### Fixed

- Fix keyboard navigation (#375)
```

Reply:
Oops, just fixed